Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.t
 # copy bot binary
 COPY --from=builder /build/target/release/turto .
 COPY --from=builder /build/config.toml.template ./config.toml
-COPY --from=builder /build/help.toml.template ./help.toml
-COPY --from=builder /build/templates.toml.template ./templates.toml
+COPY --from=builder /build/help.toml.zh-Hant.template ./help.toml
+COPY --from=builder /build/templates.toml.zh-Hant.template ./templates.toml
+COPY --from=builder /build/.env ./.env
```

Comment:
Here is a typo `/build/.env` &rarr; `/build/.env.template`

Reply:
I forgot I changed these for testing. There is a .env file in my local system.
Maybe remove this line to prevent it from overriding the TOKEN passed in? I am not sure about the behavior.